site/app/templates/admin/Docker.twig

```diff
@@ -142,10 +145,16 @@
                         <span class="badge badge-{{ capability_to_color[capability] % 8 + 1 }}">{{capability}}</span>
                     {% endfor %}
                 </td>
+                <td>
+                    {% if (docker_image_owners[key] is defined and user_id == docker_image_owners[key]) or superuser %}
+                    <button class="send-button btn btn-default" id="{{ key }}" data-image-id = "{{ key }}" onclick=confirmationdialog("{{admin_url}}/remove_image",id)>Remove</button>
```

Comment:
Why is the admin_url variable required?

Reply:
I don't know if it's required per se, but this is how adding and updating images are handled in this file so I assume it was copied from there.